framework/assets/yii.validation.js

```diff
@@ -58,15 +58,17 @@ yii.validation = (function ($) {
                 return;
             }
 
+            if (options.is !== undefined && value.length != options.is) {
```

Comment:
`is` conflicts with `min` and `max` and it does not make sense to use these options together. We either check for exact value or `min` or `max`. So it's better to skip `min` and `max` checks if `is` option is specified and check it first.